### DISCUSSION TITLE
Feature: Rotsprite

### DISCUSSION THREAD

**Discussion by msmshazan** [Orama-Interactive/Pixelorama#563] [Ideas]
Implement rotsprite for sprite rotation 
See https://forums.sonicretro.org/index.php?threads/sprite-rotation-utility.8848/ for implementation details

**Comment by OverloadedOrama** [comment]
Hello. The link you attached is old and the download link doesn't seem to work, so I'm not sure if this can be used. I'll do some more research to see if I can find another algorithm for rotating sprites, to avoid re-inventing the wheel.

**Comment by msmshazan** [comment]
Oh sorry here's the updated link with the utility http://info.sonicretro.org/RotSprite

**Comment by msmshazan** [comment]
I also found the Wikipedia article describing it https://en.m.wikipedia.org/wiki/Pixel-art_scaling_algorithms#RotSprite

**Comment by OverloadedOrama** [comment]
I took a look. This looks very interesting, but a little advanced for now. I'll definitely give it a try in the future though!
